pages/vi/profiles/jamajama1.md

```diff
@@ -0,0 +1,28 @@
+# Introduction
+### Name
+Jama Jama
+### Location
+Buffalo, NY
+### Timezone
+NY Eastern Time
+### Operating System
+Microsoft Windows 10 Home edition.
+
+---
+
+![alt text](http://arts-sciences.buffalo.edu/content/arts-sciences/linguistics/about/linguistics-at-ub/jcr:content/par/image.img.688.auto.q80.jpg/1506973707788.jpg "University at Buffalo")
+
+# About Me
+
+I am a software engineer attending University at Buffalo at the School of Engineering and Applied Science. 
+I am working on improving my skills as an engineer and currently pursuing internship opportunities. 
+I consider myself a web scraping expert. I hope to work with you all in the near future. You can find my 
+LinkedIn page [here](https://www.linkedin.com/in/jama-jama-007915162). Feel free to connect with me.
+
+Languages I have worked with:
```

Comment:
Hi @jamajama1. Everything looks good, except the list:

![Screen Shot 2019-04-16 at 5 41 18 PM](https://user-images.githubusercontent.com/35533801/56253452-3cd9fa00-6071-11e9-90a1-85691520c810.png)
To fix that you could define `Languages I have worked with:` with a heading markup ( `## Languages I have worked with: `) or add an empty line after that phrase.